content/docs/csidriver/installation/helm/powerstore.md

```diff
@@ -229,6 +228,7 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | controller.vgsnapshot.enabled | To enable or disable the volume group snapshot feature | No | "true" |
 | images.driverRepository | To use an image from custom repository | No | dockerhub |
 | version | To use any driver version | No | Latest driver version |
+| allowAutoRoundOffFilesystemSize | Allows the controller to round off filesystem to 3Gi which is the minimum supported value | No | false |
```

Comment:
1. filesystem -> NFS filesystem
2. Can we mention "which is the minimum supported size of NFS volume" ?